internal/datacoord/meta.go

```diff
@@ -302,32 +330,32 @@ func (m *meta) UpdateFlushSegmentsInfo(
 		zap.Any("check points", checkpoints),
 		zap.Any("start position", startPositions),
 		zap.Bool("importing", importing))
+	m.Lock()
+	defer m.Unlock()
+
 	segment := m.segments.GetSegment(segmentID)
 	if importing {
 		m.segments.SetRowCount(segmentID, segment.currRows)
 		segment = m.segments.GetSegment(segmentID)
 	}
 	if segment == nil || !isSegmentHealthy(segment) {
+		log.Warn("meta update: update flush segments info - segment not found")
```

Comment:
Kind of vague about why segment not found. Shall we put reason in log? Segment nil or segment not healthy?

Reply:
Also segment id is needed here.

Reply:
Done.